Content.Server/Anomaly/AnomalySynchronizerSystem.cs

```
@@ -40,6 +43,43 @@ public override void Initialize()
         SubscribeLocalEvent<AnomalyStabilityChangedEvent>(OnAnomalyStabilityChanged);
     }
 
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        var query = EntityQueryEnumerator<AnomalySynchronizerComponent, TransformComponent>();
+        while (query.MoveNext(out var uid, out var sync, out var xform))
+        {
+            if (sync.ConnectedAnomaly is null)
+                continue;
+
+            if (_timing.CurTime < sync.NextCheckTime)
+                continue;
+            sync.NextCheckTime = _timing.CurTime + sync.CheckFrequency;
+
+            if (!EntityManager.EntityExists(sync.ConnectedAnomaly))
+            {
+                DisconnectFromAnomaly((uid, sync), sync.ConnectedAnomaly.Value);
+                continue;
+            }
```

Null above is swallowing this error.